packages/marketplace/src/components/pages/login.tsx

```diff
@@ -48,8 +48,7 @@ export const tabConfigs = ({ loginType, history }: LoginProps): TabConfig[] => [
 ]
 
 export const Login: React.FunctionComponent<LoginProps> = (props: LoginProps) => {
-  const reapitEnv = process.env.REAPIT_ENV || 'LOCAL'
-  const isReapitEnvProd = reapitEnv === 'PROD'
+  const isProduction = window.reapit.config.appEnv === 'production'
```

Comment:
In this case we need to differentiate between dev and prod - only prod hides the login options tabs

Reply:
yes only production hide login option tabs